src/marinade.ts

```diff
@@ -148,4 +148,45 @@ export class Marinade {
       transactionSignature,
     }
   }
+
+  async liquidUnstake (amountLamports: BN): Promise<MarinadeResult.LiquidUnstake> {
+    const ownerAddress = this.config.wallet.publicKey
+    const marinadeState = await this.getMarinadeState()
+    const transaction = new web3.Transaction()
+
+    const {
+      associatedTokenAccountAddress: associatedMSolTokenAccountAddress,
+      createAssociateTokenInstruction,
+    } = await getOrCreateAssociatedTokenAccount(this.anchorProvider, marinadeState.mSolMintAddress, ownerAddress)
+
+    if (createAssociateTokenInstruction) {
+      transaction.add(createAssociateTokenInstruction)
+    }
+
+    const depositInstruction = await this.marinadeProgram.instruction.liquidUnstake(
```

Comment:
rename to `const liquidUnstakeInstruction`

Reply:
Fixed